### ISSUE TITLE
Framework-specific dependencies in project.json are being ignored

### ISSUE THREAD

**Issue by am11** [OmniSharp/generator-aspnet#568]
When we run:

```sh
yo aspnet:UserSecrets
```

The secrets generator populates `project.json` with:

```json
"userSecretsId": "aspnet5-<my-app>-<GUID>",
"dependencies": {
  "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-rc1-final"
}
```

It will still add this dependency, even if we have it listed under framework dependencies:

```json
"frameworks": {
    "dnx452": {
      "dependencies": {
        "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-rc1-final",
         " .. other packages .. "
      }
    }
}
```

Not a major issue, but would be nice to consider *any* dependency when checking for the existing list of packages.

**Comment by peterblazejewicz** [comment]
@am11 
This will be fixed in #570 
Thanks!

**Comment by am11** [comment]
Thanks @peterblazejewicz! :tada:
